src/webshare.js

```diff
@@ -164,7 +164,9 @@ const shouldIncludeResult = (item, showInfo) => {
   if (
     showInfo.type == "movie" &&
     item.SeasonEpisode &&
-    !item.name.toLowerCase().includes("part")
+    ["part", "episod"].every(
```

Comment:
Tady by mělo být `episode`?

Reply:
Hmmm, tak to mi nepřijde jako dobrý příklad - neřeší se příčina ale náhodné slovo, které se  v názvu taky objevuje. Jako můžeme to tam klidně nechat, ale toto není dobrý důvod.